### PR TITLE
Created August Monthly Update

### PR DESCRIPTION
**Pull request by rubymorillo**
@indirect here's the monthly update for August. I've left a few comments in the file in places where we need more content.

### CODE REVIEW

File: app/views/news/2018-09-06-august-2018-monthly-update.html.md [hunk 1]
```diff
@@ -0,0 +1,72 @@
+---
+summary: In August, we paid for 113 hours of developer work, shipped a new Bundler guide, and fixed bugs in Bundler and RubyGems.
```
Comment (rubymorillo) [inline app/views/news/2018-09-06-august-2018-monthly-update.html.md]: > summary: In August, we paid for 113 hours of developer work, shipped a new Bundler guide, and fixed bugs in Bundler and RubyGems.

@indirect feel free to update ^

File: app/views/news/2018-09-06-august-2018-monthly-update.html.md [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+
+In August, Ruby Together was supported by 74 different companies. [Handshake](https://www.handshake.org) joined as a [LEVEL] member. In total, we were supported by 77 individual members and 64 friends of Ruby Together. Thanks to all of our members for making everything that we do possible.
+
+In community events, our developer evangelist PJ Hagerty spent some time at the Google Developer Group meetup in New York City, Scenic City Summit in Chattanooga, TN, and AmsterdamRB. Executive director André Arko went to RailsCamp to spread the word about Ruby Together. 
+
+In other community news, our Google Summer of Code students wrapped up their work on our various projects. Special thanks to our GSoC developers for all of their hard work these past few months!
+
+## bundler news
+
+And with the help of [@eanlain](github.com/eanlain), we shipped a new guide: [“How to use Bundler with Docker”](https://bundler.io/v1.16/guides/bundler_docker_guide.html).
```
Comment (rubymorillo) [inline app/views/news/2018-09-06-august-2018-monthly-update.html.md]: > bundler news

@indirect I imagine there were way more things that happened in Bundler aside from the Docker guide. I structured it so we'd open with whatever important changes happened last month, and they'd end with the guide and w/the announcement of me joining the core team. Let me know if that works!

File: app/views/news/2018-09-06-august-2018-monthly-update.html.md [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+This month, Bundler gained 58 new commits, contributed by 9 authors. There were 334 additions and 54 deletions across 30 files.
+
+## rubygems.org news
+
+In August, we blacklisted gem namespaces which had derivative spellings of Rails dependencies in response to CVE-2018-3779. We also made performance improvements to the “rubygems#show” and “version#index” pages.
+
+This month, RubyGems.org gained 7 commits from 2 authors, making 45 additions and 36 deletions across 9 files.
+
+## rubygems news
+
```
Comment (rubymorillo) [inline app/views/news/2018-09-06-august-2018-monthly-update.html.md]: > rubygems news

@indirect I know Luis worked on a bunch of stuff for RG last month; maybe we can highlight some of those bits?

File: app/views/news/2018-09-06-august-2018-monthly-update.html.md [hunk 4]
```diff
@@ -0,0 +1,94 @@
...
+## rubygems.org news
+
+In August, we blacklisted several gems with names that were close to other popular gems, in response to CVE-2018-3779. We're starting to investigate ways to protect RubyGems.org from malicious gems—if you're interested in helping work on that, let us know! We also reviewed and merged performance improvements to the “rubygems#show” and “version#index” pages, contributed by [@nateberkopec](https://github.com/nateberkopec).
+
+In total, RubyGems.org gained 11 commits from 5 authors, making 44 additions and 35 deletions across 8 files.
+
+## rubygems news
+
+In RubyGems, we fixed some bugs, fixed some tests, and made sure that gems with `allowed_push_host` set will be pushed to the correct host by `gem push`. In total there were 19 new commits, contributed by 5 authors, with 112 additions and 26 deletions across 13 files.
+
```
Comment (bronzdoc) [inline app/views/news/2018-09-06-august-2018-monthly-update.html.md]: Maybe we can add this too? [Fix auto resign expired certificate](https://github.com/rubygems/rubygems/pull/2380)

Comment (rubymorillo) [reply app/views/news/2018-09-06-august-2018-monthly-update.html.md]: Sounds good! Adding it in now.

**Pull request merged**